### PR TITLE
Source Hubspot: remove FeedbackSubmissions stream

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/oncall/issues/586
Removing the stream as the one using unstable ([beta](https://developers.hubspot.com/docs/api/crm/feedback-submissions)) API and causing 500 http errors we can not fix

### CODE REVIEW

**Review by lazebnyi — APPROVED** [review]

**Review by arsenlosenko — APPROVED** [review]

File: airbyte-integrations/connectors/source-hubspot/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_hubspot ./source_hubspot
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.83
+LABEL io.airbyte.version=0.1.84
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-hubspot/Dockerfile:37]: Please bump to a major version if you are introducing a backward incomaptible change

File: airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml [hunk 2]
```diff
@@ -15,18 +15,20 @@ tests:
...
   discovery:
     - config_path: "secrets/config.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.1.83"
   basic_read:
     - config_path: "secrets/config.json"
       timeout_seconds: 600
       configured_catalog_path: "sample_files/basic_read_catalog.json"
-      empty_streams: ["form_submissions", "ticket_pipelines", "engagements_meetings", "engagements_emails", "engagements", "feedback_submissions", "engagements_calls", "quotes"]
+      empty_streams: ["form_submissions", "ticket_pipelines", "engagements_meetings", "engagements_emails", "engagements", "engagements_calls", "quotes"]
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-hubspot/acceptance-test-config.yml:24]: @YowanR, these streams are empty on our sandbox account it means they are not tested properly covered by SAT and might not work as expected in production. Do you think you could create and prioritize the provisionning of our test account with some data on this stream so we can keep the wolf from the door 😄

File: airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py [hunk 3]
```diff
@@ -1528,11 +1528,12 @@ class EngagementsTasks(CRMSearchStream):
...
-    entity = "feedback_submissions"
-    associations = ["contacts"]
-    primary_key = "id"
-    scopes = {"crm.objects.feedback_submissions.read"}
+# this stream uses a beta endpoint thus is unstable and disabled
+# class FeedbackSubmissions(CRMObjectIncrementalStream):
+    # entity = "feedback_submissions"
+    # associations = ["contacts"]
+    # primary_key = "id"
+    # scopes = {"crm.objects.feedback_submissions.read"}
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py:1536]:
```suggestion
class FeedbackSubmissions(CRMObjectIncrementalStream):
    entity = "feedback_submissions"
    associations = ["contacts"]
    primary_key = "id"
    scopes = {"crm.objects.feedback_submissions.read"}
```

I'd suggest keeping this uncommented and rather comment it in `SourceHubspot.stream` function.

**Review by alafanechere — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-hubspot/Dockerfile (hunk 1 above)
Comment (davydov-d) [reply airbyte-integrations/connectors/source-hubspot/Dockerfile]: good point. done

**Review by alafanechere — APPROVED** [review]
Thank you for the changes. Feel free to merge if the tests are passing.

**Pull request merged**